src/distributions/dragonwell/installer.ts

```diff
@@ -150,7 +150,17 @@ export class DragonwellDistribution extends JavaBase {
         // Some version of Dragonwell JDK are numerated with help of non-semver notation (more then 3 digits).
         // Common practice is to transform excess digits to the so-called semver build part, which is prefixed with the plus sign, to be able to operate with them using semver tools.
         if (jdkVersion.split('.').length > 3) {
```

Comment:
@Accelerator1996 - why not putting just one line before line 152 with `jdkVersion = jdkVersion.split('+').join('.')` ?
That would replace any `+` with a `.` upfront and parsing can continue as usual.

Reply:
> @Accelerator1996 - why not putting just one line before line 152 with `jdkVersion = jdkVersion.split('+').join('.')` ? That would replace any `+` with a `.` upfront and parsing can continue as usual.

Thank you very much for your advice. I have optimized this part of the code.

Reply:
Hello @Accelerator1996, To address the Dragonwell issue, could you please update only the following line in installer.ts as suggested by @mhoffrog:?
jdkVersion = jdkVersion.split('+').join('.');
This change will resolve the parsing issue of Dragonwell versions with the existing test data and test cases, as confirmed by our test workflow runs. It ensures that the Java version strings are correctly formatted for Dragonwell without the need for additional test data or test cases.

Reply:
> Hello @Accelerator1996, To address the Dragonwell issue, could you please update only the following line in installer.ts as suggested by @mhoffrog:? jdkVersion = jdkVersion.split('+').join('.'); This change will resolve the parsing issue of Dragonwell versions with the existing test data and test cases, as confirmed by our test workflow runs. It ensures that the Java version strings are correctly formatted for Dragonwell without the need for additional test data or test cases.

Hello @aparnajyothi-y , thank you very much for your suggestion. I have further simplified the code and removed the original if judgment, because I think the version number of dragonwell guarantees that this step can be removed. In addition, I am very sorry that I cannot just perform the replace operation as @mhoffrog suggested, because dragonwell has added two new version numbers belonging to its own products based on the upstream jdk version number, and directly calling convertVersionToSemver cannot parse.